tools/system_libs.py

```diff
@@ -852,13 +868,22 @@ class libcxxabi(CXXLibrary, NoExceptLibrary, MTLibrary):
       '-D_LIBCXXABI_GUARD_ABI_ARM',
     ]
 
+  def get_depends(self):
+    if self.eh_mode == 'wasm':
+      return ['libc', 'libunwind']
```

Comment:
Is libunwind normally linked separately like this or included in another library?

Reply:
It is a separate library, but it is only used with libcxxabi.